junifer/data/coordinates.py

```diff
@@ -134,9 +136,27 @@ def load_coordinates(name: str) -> Tuple[ArrayLike, List[str]]:
         The coordinates.
     list of str
         The names of the VOIs.
+
+    Warns
+    -----
+    DeprecationWarning
+        If ``Power`` is provided as the ``name``.
+
     """
     if name not in _available_coordinates:
         raise_error(f"Coordinates {name} not found.")
+
+    # Put up deprecation notice
+    if name == "Power":
```

Comment:
I was thinking to use something like this: https://deprecation.readthedocs.io/en/latest/

Reply:
while encapsulating the "power" in a function so we can keep track of the deprecation cycle. Otherwise we need to memorise stuff.

Reply:
This would require us to include another dependency for junifer and for one release cycle I don't think it's worth it. I think this makes sense once we start having objects which require backwards incompatible changes and that would happen after v1.0.0 imo.

Reply:
We'll create another PR for that. I don't want to have this messages all around. And this will happen quite often i'm afraid.